src/main/jbake/content/blog/2017/03/15/hawkular-opentracing-otsc.adoc

```diff
@@ -0,0 +1,61 @@
+= Distributed Tracing Workshop and OpenTracing Collaboration
+Pavol Loffay
+2017-03-15
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog
+
+The distributed tracing workshop is organized by the
+https://groups.google.com/forum/#!forum/distributed-tracing[Distributed Tracing Workgroup]
+and takes place approximately once a year. This time it was 18th-19th February in Berlin. It was a pleasure
+to be there with people from various companies and share ideas and a common goal to improve distributed tracing.
+
+== Agenda
+The agenda covered various hot topics like in-process context propagation, proposal for trace context header, state
+of OpenZipkin ecosystem and the latest happenings in OpenTracing. There were also research presentations by
+Jonathan Mace from Brown University about Baggage protocol and Baggage buffers. Some part of the workshop
+were product presentations by:
+
+* Instana
+* Datadog
+* Amazon X-Ray
+* Jaeger by Uber
+
+It was great to see how other projects present data to end users and what type of higher level analysis they focus on.
+
+== Summary
+
+We discussed many things, and here are my main takeaways:
```

Comment:
Still think colon at the end here doesn't look right when the two points below similarly have a colon at the end. Needs to be something that distinguishes this first sentence from the two points.